examples/token-sale/token_sale.yml

```diff
@@ -1,5 +1,6 @@
 name: "My awesome token"
 supportedstandards: ["NEP-17"]
+safemethods: ["balanceOf", "decimals", "symbol", "totalSupply"]
```

Comment:
Don't we also need to add safe methods to the other examples?

Reply:
We may add, but I see no much value in doing this. This example is primarily for `NEP-17` where this `Safe` is required.